torch/sparse/semi_structured/__init__.py

```diff
@@ -0,0 +1,79 @@
+import torch
+from .semi_structured_sparse_tensor import SemiStructuredSparseTensor
+import random
+
+__all__ = [
```

Comment:
We also need to update the docs for [our website](https://pytorch.org/docs/master/) with these strings.

I might even suggest we move all this into torch/sparse only and add a section on the semi structured layout to [existing documentation}(https://pytorch.org/docs/master/sparse.html)?

The high level story for this layout is not different from that of other layouts. It just happens to be supported very well on certain NVIDIA GPUs.

Reply:
Sounds good, I like the idea of having it top level instead of inside `semi_structured`